core/modules/views/tests/views_upgrade.test

```diff
@@ -13,10 +13,11 @@ require_once BACKDROP_ROOT . '/core/modules/views/tests/views_query.test';
  * You can find all conversions by searching for "moved to".
  */
 class ViewsUpgradeTestCase extends ViewsSqlTest {
-  protected function setUp() {
-//     // To import a view the user needs use PHP for settings rights, so enable php module.
-    parent::setUp();
+  protected function setUp(array $modules = array()) {
+    parent::setUp($modules);
 
+    // To import a view the user needs use PHP for settings rights, so enable php module.
+    // @TODO The php module does not exist in core in Backdrop!
     module_enable(array('php'));
```

Comment:
Should we open a follow-up for that? It's unrelated here, but looks like an oversight.

Reply:
Also there is no Views-specific importer any more, we just use the config-import to import views now. Seems this line should be able to be removed? It appears this just fails silently when a module is not found.

Reply:
New issue opened: https://github.com/backdrop/backdrop-issues/issues/5620

Someone with proper permissions can mark this comment as "resolved".

Reply:
Thank you, @indigoxela  and @quicksketch.